src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -52,11 +61,32 @@ public Climber() {
     m_rightPIDController.setI(ClimberConstants.kPID.i(), 0);
     m_rightPIDController.setD(ClimberConstants.kPID.d(), 0);
     m_rightPIDController.setFF(0);
+    m_state = State.PRE_FIRST_USE;
+  }
+
+  private boolean canMoveClimber(double position, double speed) {
+    if (
+      m_state == State.AFTER_FIRST_USE &&
+      position <= ClimberConstants.kMinRetractedLength &&
+      Math.signum(speed) == -1.0 ||
```

Comment:
`< 0.0` is IMO clearer here. If you want to go further with abstraction, you can build an enum with `EXTEND`\'RETRACT` values and use that in this boolean expression, but that probably isn't worth the trouble for a subsystem of this relatively low complexity.